ui/src/main/resources/GoogleApps/GoogleAppsConfigClass.js

```diff
@@ -1,115 +1,119 @@
 require(['jquery'], function (jQuery) {
 
-    var prefix = '#GoogleApps\\.GoogleAppsConfigClass_0_';
-
-    //deactivating
-    function deactivate(elts) {
-        elts.each(function () {
-            jQuery(jQuery(this).closest('dl')).find('label').css('color','darkgrey');
-            this.wasDisabled = jQuery(this).prop('disabled');
-            jQuery(this).prop('disabled', true);
-        });
+  var prefix = '#GoogleApps\\.GoogleAppsConfigClass_0_';
+
+  //deactivating
+  function deactivate(elts) {
```

Comment:
``elts`` is a really bad name for ``elements``.

Reply:
changed.